PULL_REQUEST_TEMPLATE.md

```diff
@@ -55,14 +55,13 @@ If you are changing components or packages in DC/OS (e.g. you are bumping the sh
 
 **What is the review process and when will my changes land?**
 
-All PRs require 2 approvals using GitHub's pull request reviews. 
+All PRs require approvals from any two users on the most recent commit. Any commits added to the pull request after approval will invalidate the approvals.
 
 Reviewers should be:
 
 * Developers who understand the code being modified.
-* Developers responsible for code that interacts with or depends on the code being modified.
```

Comment:
I think the original makes more sense:
Reviewers should be: "Developers responsible for <any> code that interacts..." NOTE: <any> is a suggested addition
vs.:
Reviewers should be "Developers are responsible for the code..."

Reply:
@pcrews  - Yeah, I agree. I will revert this statement back.